geomesa-accumulo/geomesa-accumulo-spark/src/main/scala/org/locationtech/geomesa/spark/accumulo/AccumuloSpatialRDDProvider.scala

```diff
@@ -48,11 +48,8 @@ class AccumuloSpatialRDDProvider extends SpatialRDDProvider {
     val ds = DataStoreFinder.getDataStore(dsParams).asInstanceOf[AccumuloDataStore]
     val username = AccumuloDataStoreParams.userParam.lookUp(dsParams).toString
     val password = new PasswordToken(AccumuloDataStoreParams.passwordParam.lookUp(dsParams).toString.getBytes)
-    try {
-      // get the query plan to set up the iterators, ranges, etc
-      lazy val sft = ds.getSchema(query.getTypeName)
-      lazy val qp = AccumuloJobUtils.getSingleQueryPlan(ds, query)
 
+    def queryPlanToRDD(sft: => SimpleFeatureType, qp: => AccumuloQueryPlan, conf: Configuration) = {
```

Comment:
No, that's what the "extract method" in IntelliJ generated, presumably because those two variables were defined as ``lazy``.